.travis.yml

```diff
@@ -7,16 +7,16 @@ env:
     - IPFS_REUSEPORT=false
 
 go:
-    - 1.8
-    - 1.9
+    - 1.8.x
+    - 1.9.x
 
 install: true
 
-script:
-  - ulimit -n 2048
+before_install:
```

Comment:
Note: I removed `ulimit -n 2048` as it doesn't seam to be necessary. If there was a good reason for it I will add it back.

Reply:
Not really. That may have been necessary to get this to pass on osx (but we no longer test it on osx).